runtime.md

```diff
@@ -90,18 +94,27 @@ Attempting to start an already started container MUST have no effect on the cont
 `run <container-id>`
 
 This operation MUST generate an error if it is not provided the container ID.
+Attempting to run a container that does not exist MUST generate an error.
 This operation MUST invoke the `create` operation, and if there are no errors, then invoke the `start` operation.
 The implementation MAY immediately execute `start` without any pause after `create` completes.
 For example if the implementation used a temporary process to implement `create`, it need not do this in the implementation of `run`.
 
+### Stop
+`stop <container-id>`
+
+This operation MUST generate an error if it is not provided the container ID.
+Attempting to stop a container that is not running MUST have no effect on the container and MUST generate an error.
+This operation MUST stop all of the processes running within the scope of the container.
+This operation MUST NOT delete any resources associated with the container, except for the PID namespace.
```

Comment:
In a runtime based on Linux namespaces, this is going to require the runtime to take steps (e.g. bind mounting) to preserve namespaces after the container process dies.  I'd rather keep discussion of that sort of thing in #356 and #395, and not make such an important change in this PR (splitting create/start is an important enough change on its own ;).